src/odemis/driver/test/xt_client_test.py

```diff
@@ -915,6 +916,181 @@ def test_beamlet_index_info(self):
+
+        # Test if errors are produced when a value outside of the range is set.
+        with self.assertRaises(IndexError):
+            self.scanner.beamStigmator.value =  tuple(1.2 * i for i in self.scanner.beamStigmator.range[1])
+        self.assertEqual(test_stigmator_value, tuple(self.microscope.get_primary_stigmator()))
+
+        # Test if the value is automatically updated when the value is not changed via the VA
+        self.microscope.set_primary_stigmator(0, 0)
+        time.sleep(6)
+        self.assertEqual((0, 0), tuple(self.scanner.beamStigmator.value))
+
+        self.scanner.beamStigmator.value = current_value
+
+    def test_pattern_stigmator_VA(self):
+        current_value = self.scanner.patternStigmator.value
+
+        # Test if directly changing it via the VA works
+        for test_stigmator_value in self.scanner.patternStigmator.range:
+            self.scanner.patternStigmator.value = test_stigmator_value
+            self.assertEqual(test_stigmator_value, tuple(self.microscope.get_secondary_stigmator()))
+
+        # Test if errors are produced when a value outside of the range is set.
+        with self.assertRaises(IndexError):
+            self.scanner.patternStigmator.value = tuple(1.2 * i for i in self.scanner.patternStigmator.range[1])
+        self.assertEqual(test_stigmator_value, tuple(self.microscope.get_secondary_stigmator()))
+
+        # Test if the value is automatically updated when the value is not changed via the VA
+        self.microscope.set_secondary_stigmator(0, 0)
+        time.sleep(6)
+        self.assertEqual((0, 0), tuple(self.scanner.patternStigmator.value))
+
+        self.scanner.patternStigmator.value = current_value
+
+    def test_beam_shift_transformtion_matrix_VA(self):
+        beamShiftTransformtionMatrix = self.scanner.beamShiftTransformtionMatrix
+        self.assertIsInstance(beamShiftTransformtionMatrix.value, list)
+        self.assertEqual(len(beamShiftTransformtionMatrix.value), 4)
+        for row_transformationmatrix in beamShiftTransformtionMatrix.value:
+            self.assertIsInstance(row_transformationmatrix, list)
+            self.assertEqual(len(row_transformationmatrix), 2)
+            self.assertIsInstance(row_transformationmatrix[0], float)
+            self.assertIsInstance(row_transformationmatrix[1], float)
+
+        # Check if VA is read only
+        with self.assertRaises(NotSettableError):
+            self.scanner.beamShiftTransformtionMatrix.value = beamShiftTransformtionMatrix
+
+    def test_multiprobe_rotation_VA(self):
+        mpp_rotation = self.scanner.multiprobeRotation.value
+        self.assertIsInstance(mpp_rotation, float)
+        # Currently the range of the value can be quite big due to different designs for microscopes.
+        self.assertGreaterEqual(mpp_rotation,  - numpy.rad2deg(90))
+        self.assertLessEqual(mpp_rotation, numpy.rad2deg(90))
```

Comment:
Please use `math.radians()` here too. No need to use numpy (and get special numpy floats) when standard functions can do the job ;-)